test/game_statistics_test.rb

```diff
@@ -0,0 +1,61 @@
+require "./lib/game_statistics"
```

Comment:
@ajtran303 I added all the code we wrote for game exactly how it was written in the game_stats branch. Still passes all the tests